src/condor_utils/file_transfer.cpp

```diff
@@ -6576,44 +6574,44 @@ FileTransfer::InvokeMultipleFileTransferPlugin( CondorError &e,
 		plugin_args.AppendArg( "-upload" );
 	}
 
-	// Invoke the plugin
-	dprintf( D_ALWAYS, "FILETRANSFER: invoking: %s \n", plugin_path.c_str() );
```

Comment:
Removing this line breaks test_curl_plugin.  I have no preference between adding this line back in (as D_TEST?) or changing the test.